.github/workflows/pr-builder.yml

```diff
@@ -38,6 +38,15 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
+    
+    - name: Install dependencies
```

Comment:
> IMO, the proper fix here would be to remove the prebuild script that installs packages and do the install explicitly.

Removing the prebuild script would cause inconsistent lock files being used to package the release artifact, wouldn't it?